### PR TITLE
Spooky Mommi for Halloween

### PR DESCRIPTION
**Pull request by BurntDevil**
Some mommi content for Halloween. Pick it like any other skin.
:cl:
 * rscadd: Spooky MoMMIs have arrived on station.

### CODE REVIEW

**Review by BurntDevil — COMMENTED** [review]
Ah, thats pretty simple, I see.

**Comment by pedr69** [comment]
image

https://user-images.githubusercontent.com/18052467/199114058-71c89c3e-72c6-46b8-bba4-0604621fdcb4.mp4

File: code/modules/mob/living/silicon/mommi/mommi_modules.dm [hunk 1]
```diff
@@ -36,6 +36,9 @@
 	sensor_augs = list("Mesons", "Disable")
 
 	fix_modules()
+	
+	if(Holiday == HALLOWEEN)
```
Comment (jknpj) [inline code/modules/mob/living/silicon/mommi/mommi_modules.dm:40]: Not sure if this is the best place for this

Comment (Kurfursten) [reply code/modules/mob/living/silicon/mommi/mommi_modules.dm:40]: where do you feel it is best?

**Pull request merged**